src/internal/useDeepCompareMemo.ts

```diff
@@ -1,10 +1,10 @@
-import { InputIdentityList, useRef } from "react";
+import { useRef } from "react";
 
 import { isDeepEqual } from "./DataUtils";
 
 export function useDeepCompareMemo<T>(
   factory: () => T,
-  inputs: InputIdentityList,
```

Comment:
@JeremyDurnell, "InputIdentityList" does not exist in react anymore, it breaks the build.